### ISSUE TITLE
Warn user to uninstall git-duet

### ISSUE THREAD

**Issue by dideler** [findmypast-oss/git-mob#2]
Because git-duet also has a git-solo command, so there will be a conflict and whichever script is present last in the PATH will be used.

**Comment by dideler** [comment]
If you see output like this, it probably means there's a naming conflict with another git-solo script (e.g. git-solo ships with git-duet), and the other script appears in your PATH after git-mob.

```
↪ git solo -v
0.5.2 ('aed36fb704bf19059e6177e02bff81cb4e6deddd')
```

**Comment by dideler** [comment]
If installed via brew, `brew uninstall git-duet`.

If manually installed via downloaded executable, delete the executables and remove its directory from your PATH if needed.

**Comment by dideler** [comment]
Resolved in c99027cbf55b811d695def896c79da4363723f75
